packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -221,6 +222,9 @@ class TextSelectionOverlay {
   /// Creates an object that manages overly entries for selection handles.
   ///
   /// The [context] must not be null and must have an [Overlay] as an ancestor.
+  ///
+  /// 'selectionStartInView' and 'selectionEndInView' are optional and can be
```

Comment:
I noticed that the `_TextSelectionHandleOverlayState` has direct access to the `RenderEditable`, so I've removed these callbacks and typedefs since it's not needed to thread this through `EditableTextState` and `TextSelectionOverlay`.

Reply:
Looks like the documentation wasn't removed at the same time